scala3doc/src/dotty/dokka/transformers/SealedMarksGraphTransformer.scala

```diff
@@ -0,0 +1,26 @@
+package dotty.dokka
+
+import org.jetbrains.dokka.transformers.documentation.DocumentableTransformer
+import org.jetbrains.dokka.model.{ Modifier => _, _ }
+import collection.JavaConverters._
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.model.properties._
+
+import dotty.dokka.model._
+import dotty.dokka.model.api._
+
+
+class SealedMarksGraphTransformer(using context: DocContext) extends ModuleTransformer:
+  override def apply(original: DModule): DModule =
+    val sealedRelations = getSealedRelations(original.getPackages.get(0))
+    original.updateMembers { m =>
+      m.mapGraphEdges {
+        case Edge(from @ LinkToType(_, fromDri, _), to @ LinkToType(_, toDri, _), _) =>
```

Comment:
can we avoid reallocation of all `Edge objects` e.g.:
```
   m.mapGraphEdges {
        case edge @ Edge(from, to, _) =>
          val desc = if sealedRelations.contains(to.dri) then Edge(from, to, "🔒") else edge
      }
```

Reply:
Yeah, my bad